include/moqt/common.h

```diff
@@ -0,0 +1,75 @@
+/*
+ *  Copyright (C) 2024
+ *  Cisco Systems, Inc.
+ *  All Rights Reserved
+ */
+
+#pragma once
+
+#include <string>
+#include <transport/transport.h>
+
+namespace moq::transport {
+
+    constexpr uint64_t kMoqtVersion = 0xff000004; ///< draft-ietf-moq-transport-04
+    constexpr uint64_t kSubscribeExpires = 0;     ///< Never expires
+    constexpr int kReadLoopMaxPerStream = 60; ///< Support packet/frame bursts, but do not allow starving other streams
+
+    using namespace qtransport;
+
+    using Bytes = std::vector<uint8_t>;
+    using BytesSpan = Span<uint8_t const>;
+
+    /**
+     * @brief Full track name struct
+     *
+     * @details Struct of the full track name, which includes the namespace tuple, name, and track alias
+     *   Track alias will be set by the Transport.
+     */
+    struct FullTrackName
+    {
+        Span<uint8_t const> name_space;
+        Span<uint8_t const> name;
+        std::optional<uint64_t> track_alias;
+    };
```

Comment:
let's make it a class and add functions . 
s/name_space/track_namespace. Would be easier of namespace was not spit.
s/name/track_name;
track_alias is not optional.